cognoml/figshare.py

```diff
@@ -0,0 +1,60 @@
+import os
+from urllib.request import urlretrieve
+import json
+
+import requests
+
+def get_article_versions(article_id):
+    """
+    Get version_to_url dictionary for a figshare article.
+    """
+    url = 'https://api.figshare.com/v2/articles/{}/versions'.format(article_id)
+    response = requests.get(url)
+    version_to_url = {d['version']: d['url'] for d in response.json()}
+    return version_to_url
+
+def download_files(directory, artile_id=3487685, version=None):
+    """
+    Download files for a specific figshare article_id and version to the specified directory.
```

Comment:
a usage note that it won't _really_ download to the specified dictionary - it will append the version too